src/functions/accent.js

```diff
@@ -180,6 +180,11 @@ const mathmlBuilder = (group, options) => {
     return node;
 };
 
+const NON_STRETCHY_ACCENT_REGEX = new RegExp([
+    "\\acute", "\\grave", "\\ddot", "\\tilde", "\\bar", "\\breve",
+    "\\check", "\\hat", "\\vec", "\\dot",
+].map(accent => `\\${accent}`).join("|"));
```

Comment:
I would have forgotten to double the backslashes here. Glad you go this right!

Reply:
Is there a reason to use a regex here instead of an object like
```js
nonstretchyAccent = {"\\acute": true, ...}
...
if (context.funcName in nonstretchyAccent)
```

In general, KaTeX should do this sort of thing in place of most/all `utils.contains`.  Faster.  But that's another issue.

In this instance, another option would be to have a mapping defining properties like `stretchy` for each accent type... could go either way. But RegExp seems slightly unnecessary (though also fine).